src/libs/export.c

```diff
@@ -480,10 +504,39 @@ static void profile_changed(GtkWidget *widget, dt_lib_export_t *d)
   dt_conf_set_string(CONFIG_PREFIX "iccprofile", "");
 }
 
-static void size_changed(GtkSpinButton *spin, gpointer user_data)
+static void width_changed(GtkEditable *entry, gpointer user_data)
 {
-  const char *key = (const char *)user_data;
-  dt_conf_set_int(key, gtk_spin_button_get_value(spin));
+  dt_lib_export_t *d = (dt_lib_export_t *)user_data;
+  _validate_dimensions(d);
+  dt_conf_set_int(CONFIG_PREFIX "width", atoi(gtk_entry_get_text(GTK_ENTRY(d->width))));
+}
+
+static void height_changed(GtkEditable *entry, gpointer user_data)
+{
+  dt_lib_export_t *d = (dt_lib_export_t *)user_data;
+  _validate_dimensions(d);
+  dt_conf_set_int(CONFIG_PREFIX "height", atoi(gtk_entry_get_text(GTK_ENTRY(d->height))));
+}
+
+static void insert_text_handler(GtkEditable *entry, char *text, int length, gpointer position, gpointer user_data)
+{
+  int i, count=0;
+  gchar *result = g_new (gchar, length);
+
+  for (i=0; i < length; i++) {
```

Comment:
I'll fix that, but please not that { should be on its own line.

Reply:
I agree. I didn't see that. I think I 'borrowed' some of this code from elsewhere.